frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+                    <Button type="submit" size="lg">
+                      Finish Backup
+                    </Button>
+                  </div>
+                </div>
+              </form>
+            </DialogContent>
+          </Dialog>
+        </div>
+      )}
+      <>
+        <Separator className="my-6" />
+        <div className="flex flex-col gap-8">
+          <div>
+            <h3 className="text-lg font-medium">Channels Backup</h3>
+            <p className="text-sm text-muted-foreground">
+              Your spending balance is stored in your lightning channels. In
+              case of recovery or migration or your Alby Hub, they need to be
+              backed up every time you open a new channel.
+            </p>
+          </div>
+
+          <div>
+            <div className="flex gap-2 mb-1 items-center">
+              <h3 className="text-sm font-medium">Automated Channels Backup</h3>
+              {info?.albyAccountConnected ? (
+                <Badge variant={"positive"}>Active</Badge>
+              ) : (
+                <Badge>Recommended</Badge>
+              )}
+            </div>
+            {info?.albyAccountConnected ? (
+              <>
+                <p className="text-muted-foreground text-sm mb-8">
+                  Your channel state is backed up automatically after each
+                  channels creation. Potential recovery will trigger channel
+                  closures, and your funds will arrive in your on-chain balance.
+                </p>
+                {info?.vssSupported && (
+                  <>
+                    <div>
+                      <div className="flex gap-2 mb-1 items-center">
+                        <h3 className="text-sm font-medium">
+                          Dynamic Channels Backup With Instant Recovery
+                        </h3>
+                        {me?.subscription.buzz && info.ldkVssEnabled ? (
+                          <Badge variant={"positive"}>Active</Badge>
+                        ) : (
+                          <Badge>Alby Cloud</Badge>
+                        )}
+                      </div>
+                      <p className="text-sm text-muted-foreground mb-4">
+                        When enabled, your channels state is dynamically updated
+                        on a virtual disk encrypted with your wallet keys
+                        (Versioned Storage Service). This allows you to
+                        immediately recover or migrate your Hub with full
+                        spending balance and without having to close your
+                        channels.
+                      </p>
+
+                      {!me?.subscription.buzz ? (
+                        <Button
+                          variant="secondary"
+                          disabled={info.ldkVssEnabled}
+                          size={"lg"}
+                          onClick={() => {
+                            if (!me?.subscription?.buzz) {
```

Comment:
why do we do the same check twice? and the component could be a `LinkButton` rather than `onClick` with a navigation. The `disabled` check also looks wrong to me here.

Note: with the DIY plan coming soon we will need to change this, I guess